modules/openshift-cluster-maximums.adoc

```diff
@@ -7,7 +7,7 @@
 
 [options="header",cols="5*"]
 |===
-| Maximum type |3.11 tested maximum |4.1 tested maximum |4.2 tested maximum |4.3 tested maximum
+| Maximum type |4.1 tested maximum |4.2 tested maximum |4.3 tested maximum |4.4 tested maximum
```

Comment:
We're no longer showing 3.11 numbers?

Reply:
@RobertKrawitz We show the latest 4 releases at a time to prevent the table from getting unruly.

Reply:
+1